src/handlers/pinnedMessages.ts

```diff
@@ -0,0 +1,128 @@
+import lineByLine from 'n-readlines'
+import { Entity, entities } from '../Entities'
+import log from '../helpers/logger'
+import { getMapping, getRoomId, getMessageId } from '../helpers/storage'
+import {
+  axios,
+  formatUserSessionOptions,
+} from '../helpers/synapse'
+
+export type PinnedMessages = { [key: string]: string[] }
+export type UserForRoom = { [key: string]: string }
+export type PromiseResult = {
+    pinnedMessages: PinnedMessages;
+    userForRoom: UserForRoom;
+};
+
+/**
+ * Type of Rocket.Chat messages
```

Comment:
I think it'd make more sense to import the type from `src/handlers/messages.ts` or put it into it's own definition file and import it from there. Or did you change anything?

Reply:
i added 
  pinnedBy?: {
    _id?: string
    username?: string
  }
But i could add it to messages.ts directly